Added in Polkadot's Papers (#733)
w3f/polkadot-wiki: docs/learn-papers.md

---
id: learn-papers

> title: Polkadot Papers

Author: I was unsure of what to actually call this page (the ID and the title), if anyone has any ideas, I would greatly appreciate them!

sidebar_label: Polkadot Papers
---

## Polkadot's White Paper

Polkadot's original white paper is a technical summary around one possible direction of implementing
the Polkadot network. This paper uses rationale and technical details to support why this direction
is beneficial. This original white paper also explains how each core component in Polkadot works
together to build this decentralized network. Polkadot's white paper can be found
[here](https://polkadot.network/PolkaDotPaper.pdf).

## Polkadot's Overview Paper

Polkadot's overview paper is a visual, easy to read, and less technical introduction into its
blockchain technology. This paper dives into the components of Polkadot but is understandable for
both a non-technical and technical reader. For reference, the overview paper can be found

> [here](https://polkadot.network/Polkadot-lightpaper.pdf).

Reviewer: So actually, the overview paper is not the same as the Light Paper.

The [overview paper](https://github.com/w3f/research/blob/master/docs/papers/OverviewPaper-V1.pdf) is an updated version of the white paper that describes the protocol in more technical terms. We should primarily suggest this paper to people who want to dig more into the protocol.

Author: Oh! Thank you for this, I'll update it. Would it also be helpful to add in the light paper as another section on this page?

Reviewer: Yes, I would suggest to keep the section the same but rename to Polkadot's Light Paper and make a new section for the Overview paper.

---
id: learn-papers
title: Polkadot Papers
sidebar_label: Polkadot Papers
---

## Polkadot's White Paper

Polkadot's original white paper is a technical summary around one possible direction of implementing
the Polkadot network. This paper uses rationale and technical details to support why this direction

> is beneficial. This original white paper also explains how each core component in Polkadot works

Reviewer 2: ```suggestion
is beneficial. This original white paper also explains how Polkadot's core components work
```

together to build this decentralized network. Polkadot's white paper can be found

> [here](https://polkadot.network/PolkaDotPaper.pdf).

Reviewer 2: Consider removing this, and linking to it in the beginning of the paragraph like you did with the overview paper. Two reasons - one is consistency, and the other is accessibility. When screen readers go through text, they just read "here" to the people when listing links, unless the link also has a title which in this case it does not. Having the link be something like `[Polkadot's white paper](url)` makes the link more accessible.

## Polkadot's Overview Paper

[Polkadot's Overview paper](https://github.com/w3f/research/blob/master/docs/papers/OverviewPaper-V1.pdf)

> is an updated version of the White paper that describes the protocol in more technical terms. We

Reviewer: ```suggestion
is an updated version of the white paper that describes the protocol in more technical terms. We
```

would recommend reading this overview paper if you are interested in digging more into the protocol
itself.

## Polkadot's Light Paper

Polkadot's light paper is a visual, easy to read, and less technical introduction into its
blockchain technology. This paper dives into the components of Polkadot but is understandable for
both a non-technical and technical reader. For reference, the light paper can be found

> [here](https://polkadot.network/Polkadot-lightpaper.pdf).

Reviewer 2: Same comment for link as above.

## The Polkadot Specification

> [Polkadot's specification](https://github.com/w3f/polkadot-spec) is a Github Repo that holds the

Reviewer 2: ```suggestion
[Polkadot's specification](https://github.com/w3f/polkadot-spec) is a Github repository that holds the
```

> latest Polkadot Host Protocol specification, Polkadot's specification tests of the many components

Reviewer: ```suggestion
latest Polkadot Host protocol specification, Polkadot's specification tests of the many components
```

> on the network, and the Polkadot Runtime specification. This repo holds algorithms and explores how

Reviewer 2: ```suggestion
of the network, and the Polkadot Runtime specification. This repo holds algorithms and explores how
```
